examples/nlp/machine_translation/create_tarred_parallel_dataset.py

```diff
@@ -116,14 +123,21 @@
         src_fname=args.src_fname,
         tgt_fname=args.tgt_fname,
         out_dir=args.out_dir,
-        encoder_tokenizer=encoder_tokenizer,
-        decoder_tokenizer=decoder_tokenizer,
+        encoder_tokenizer_name=args.encoder_tokenizer_name,
+        encoder_model_name=args.encoder_model_name,
+        encoder_tokenizer_model=encoder_tokenizer_model,
+        encoder_bpe_dropout=args.encoder_tokenizer_bpe_dropout,
+        decoder_tokenizer_name=args.decoder_tokenizer_name,
+        decoder_model_name=args.decoder_model_name,
+        decoder_tokenizer_model=decoder_tokenizer_model,
+        decoder_bpe_dropout=args.decoder_tokenizer_bpe_dropout,
         max_seq_length=args.max_seq_length,
         min_seq_length=args.min_seq_length,
         tokens_in_batch=args.tokens_in_batch,
         lines_per_dataset_fragment=args.lines_per_dataset_fragment,
         num_batches_per_tarfile=args.num_batches_per_tarfile,
-        pkl_file_prefix=args.pkl_file_prefix,
+        tar_file_prefix=args.tar_file_prefix,
```

Comment:
Should we add this variable to the yaml config as well?

Reply:
Done.